### ISSUE TITLE
Cannot find texture/toon settings in v2.8

### ISSUE THREAD

**Issue by Jatts-Art** [powroupi/blender_mmd_tools#264]
So... I know that this is most likely not an "MMD Tools" related issue, but more of a general MMD issue (regarding Blender). But I would still greatly appreciate some sort of help since I don't really know of other MMD/Blender communities except people on Deviantart (which I refuse to bother with). Due to a complicated series of problems that I am still dealing with right now, I was forced to switch to Blender 2.80 after using 2.78 for over 2 years. Struggling to get back on track with a personal project of mine, I have been wasting my time basically relearning the entire interface and many other issues but I digress.... point is, there is a particular menu that I used often regarding textures and toons that I no longer seem to have access to for materials.

The texture tab from Blender 2.78:
![b72135871db6ba91134c6d04c8b119fd](https://user-images.githubusercontent.com/12752293/74213533-b4fc5680-4c67-11ea-884f-405f7b5adee3.png)

Blender 2.80:
![5747f0045fad87aa34e5bb76f48f17bd](https://user-images.githubusercontent.com/12752293/74213559-cc3b4400-4c67-11ea-9d87-81c901bdf68c.png)

So basically I can no longer change the settings of spheres and toons from the Blender side of things anymore which I found useful for my own reasons. My question is whether there is something relating to MMD Tools that gets around this, or is there a new tab entirely I'm not aware of? I also exported another MMD model just to see if they have a texture listing for its materials and they don't either, which confuses me even more considering the textures/toons looked completely fine on that model as if those settings _are_ still intact despite being missing. I would appreciate any help, thank you for your time.

**Comment by nathanvasil** [comment]
Those kinds of textures were only ever used in the Blender Internal renderer, which no longer exists in 2.80+.  Cycles and Eevee are now your only renderer options (well, and Workbench) and they use nodes for their materials.  You can see these nodes by changing a viewport to "Shader editor."  It can take a bit to understand how nodes materials work, and it would be worthwhile to hunt down a youtube tutorial or something.

Some MMD Tools specific features, which aren't really used for rendering but are used for export, exist on properties/material, provided you have the addon installed.

DA isn't a great place to learn about Blender, but there are other Blender resources out there.  blenderartists.org is a great place for getting help on anything Blender related.  reddit.com/r/blenderhelp is a decent place as well.  There aren't a ton of people familiar with MMD Tools on either place (other than me, I answer a lot of questions on both those sites) but there are probably some, and most MMD Tools questions are really Blender questions at their heart-- MMD Tools doesn't really do anything super special or weird, it just uses existing Blender functionality.  (Well, occasional exceptions, like SDEF support.)

**Comment by Jatts-Art** [comment]
Thank you for the response and recommendations! Shortly before your answer, I eventually did learn what the issue was and it uses nodes now as you said. Good news is that everything else for Blender 2.8 is now setup for continuing my MMD work, the only issue left is this texture/node business. This leads me to my next question...

Here are my list of previous settings, but it seems they have automatically been transferred into this brush menu:
![b1891bfd1b0f13a4975850218b700385](https://user-images.githubusercontent.com/12752293/74288900-b24b4100-4cfb-11ea-9122-ff1241d9d2ec.png)

When I actually need them to be in this menu instead, sadly:
![b4ce3438ec3ce6d5244683148087e67d](https://user-images.githubusercontent.com/12752293/74288957-d149d300-4cfb-11ea-98ea-99790c77ba12.png)

And yes... I do have 161 listings there, lol. However my 161 materials only utilized 17 shader/texture settings. Does this mean I have to setup 161 nodes now? :/ Or would I only need to make 17 nodes and bridge them across? 

Due to this not being an MMD Tools related problem, if you want you could direct me to maybe some sort of guide relating to nodes and/or nodes relating to MMD and then I'll move on from there I guess? Up to you if you can of course, but either way it seems I'll have to most likely figure out the rest on my own somewhere else. I appreciate the response!

**Comment by nathanvasil** [comment]
Don't even look at the texture tab.  For most MMD users, it's currently useless.  (Still used by some things like displace modifier textures.)

161 materials is a lot, yeah.  My guess, if you're only using 17 image textures for them,  is that many of them are redundant and you could simplify.  You can assign faces to existing mats; you can use the material specials to clean unused slots; etc.  You can convert to Cycles mats via MMD Tools to quickly convert them to mats: sidebar/MMD/convert materials for Cycles.

There are plenty of techniques to make life easier and "edit it once"-- node groups are a good idea, probably.  You can copy/paste nodes, node groups, etc.

I know that most MMD users don't want to learn all of Blender to make it do what they want, but it really is the best idea.  I wouldn't recommend searching for MMD specific information.  There simply isn't a lot of MMD+Blender specific information out there.  Instead, just learn how to use Blender.  Watch Blender tutorials.  Learn your nodes.  Put tuts on in the background and fast forward, or pay attention only when they mention something you didn't know-- I know I personally hate how all the information is in video, but there's a lot of information out there.

I can't recommend any specific tutorials related to materials.  I've generally learned by experimentation.

**Comment by Jatts-Art** [comment]
Ah I see, yeah experimentation is pretty much how I've even gotten to this point. I actually begun to learn a decent amount of Blender 2.7 during these past two years or so before having to switch to 2.8, heh. An example of what I've done: https://3d.nicovideo.jp/works/td60809

But yeah copy/pasting nodes sounds like a good idea so I guess it won't be too bad! As for all those materials, its hard to explain but there's alot of reasons why I had it the way it was but I won't get into that since it isnt important. Thanks again for the help, I think I'll be good now!
